package.json

```diff
@@ -5,8 +5,7 @@
   "main": "server.js",
   "scripts": {
     "start": "nodemon ./server/",
-    "test": "node ./server/tests/routes.test.js | tap-spec",
-    "test:watch": "tape -w",
```

Comment:
Any reason you got rid of this? Doesn't really matter either way but I'd leave it in just for a reference even if it's not explicitly used. Not too bother tho tbf.

Reply:
It doesn't work

Reply:
Yeah, the `-w` flag doesn't work. You'd need to use the [tape-watch](https://www.npmjs.com/package/tape-watch) node module